core/src/main/java/nl/nn/adapterframework/senders/IbisLocalSender.java

```diff
@@ -44,10 +44,16 @@
 import nl.nn.adapterframework.util.Misc;
 
 /**
- * Posts a message to another IBIS-adapter in the same IBIS instance.
- *
- * Returns exit.code as forward name to SenderPipe.
- *
+ * Posts a message to another IBIS-adapter in the same IBIS instance. If the callee exits with an &lt;<code>exit</code>&gt;
+ * that has state {@link nl.nn.adapterframework.core.PipeLine.ExitState#ERROR}, an error is considered to happen
+ * in the caller which means that the <code>exception</code> forward is followed if it is present.
+ * <p/>
+ * <p/>
+ * Returns exit.name as forward name to SenderPipe. For example, if the called adapter has an exit state with name
```

Comment:
Niet het naam-attribuut, maar het code-attribuut wordt als forwardnaam teruggegeven.

Reply:
Naar aanleiding van je comment heb ik het nog eens gecontroleerd. Je hebt gelijk. Wel werkt het alleen als de code een string is die geparsed kan worden als integer. Dus met codes "c1" en "c2" werkt het niet, wel met codes "1" en "2".